src/manifolds/Euclidean.jl

```diff
@@ -25,6 +25,8 @@ function Euclidean(n::Vararg{Int,I}; field::AbstractNumbers = ℝ) where {I}
     return Euclidean{Tuple{n...},field}()
 end
 
+^(𝔽::AbstractNumbers, n) = Euclidean(n...; field = 𝔽)
```

Comment:
This is new, right? Neat idea!

Reply:
We had this before, just in `numbers.jl` for some reason.

Reply:
Ah I looked for it bun then I must have missed it, looks really nice, especially handy if you do a demo (and know how to type that utf8 letter fast, I'll have to check though).

Reply:
In REPL, VSCode and Juno, you can insert them by typing `\bbR`, `\bbC` and `\bbH`.